pipeline.wdl

```diff
@@ -29,7 +29,9 @@ workflow pipeline {
         Boolean detectNovelTranscipts = false
         File? cpatLogitModel
         File? cpatHex
-        File dockerTagsFile
+        File dockerImagesFile
+        # Only run multiQC if the user specified an outputDir
+        Boolean runMultiQC = if (outputDir == ".") then false else true
```

Comment:
wouldn't just `outputDir != "."` work?

Reply:
:man_facepalming: 
I fell for the age old `if bla then boolean` trap!